react-ui/src/pages/entry/EntryAppBarRight.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Entry-specific content for top app bar
+ */
+
+import FlagTwoToneIcon from '@material-ui/icons/FlagTwoTone';
+import InfoOutlinedIcon from '@material-ui/icons/InfoOutlined';
+import RedditIcon from '@material-ui/icons/Reddit';
+import PropTypes from 'prop-types';
+
+import {
+  CustomIconButton,
+} from '../../components';
+
+export default function EntryAppBarRight({
+  // eslint-disable-next-line react/prop-types
+  entry = {},
```

Comment:
We're only using `imagePath` and `permalink` here, so we can just define those. They're both strings.